sdk/maps/Azure.Maps.Search/src/MapsSearchClient.cs

```diff
@@ -518,6 +519,11 @@ private static GeocodingBatchRequestBody geocodingQueriesToGecodingBatchRequestB
                 {
                     item.BoundingBox = new GeoBoundingBox(query.BoundingBox.West, query.BoundingBox.South, query.BoundingBox.East, query.BoundingBox.North);
                 }
+                if(item.Query == item.CountryRegion)
+                {
+                    item.Query = new GeoPosition(Convert.ToDouble(query.Coordinates?.Longitude + query.Locality), Convert.ToDouble(query.Coordinates?.Latitude + query.Locality));
```

Comment:
`query.Locality` is a string for characters, not a string number. I believe this won't work.